docs/docs/quickstart/react.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: React
+hide_title: true
+---
+
+<h1 style={{ display: 'flex', gap: "2rem", alignItems: 'center' }}>Quickstart: React <img alt="npm" src="https://img.shields.io/npm/v/@simpleweb/open-format-react"/></h1>
+
+The React SDK adds a small abstraction over the [Javascript SDK](./javascript) but this simplifies interacting with Open Format even further.
+
```

Comment:
Is it worth highlighting that this is only a requirement with create react app?? I spun up a next app today and there were no issues. I appreciate this is a small point and probably a large piece of work to be doing so late in the day.